tests/core/vm/test_london.py

```diff
@@ -288,19 +292,16 @@ def test_state_does_not_revert_on_reserved_0xEF_byte_for_CREATE_and_CREATE2_pre_
 ):
     chain = pre_london_miner
     vm = chain.get_vm()
-    state = vm.state
 
-    create_message = Message(
+    computation = vm.execute_bytecode(
```

Comment:
Was this change important, beyond being shorter/cleaner?

Reply:
This seemed like a more appropriate way to test the bytecode tests provided by the EIP. It will still end up calling `apply_create_message()`. Having written the original tests I just wanted to make sure it is tested from multiple angles I suppose and I was trying to find out what would be the most appropriate instantiation (if there is one?). But it also feels a bit cleaner / shorter hah.

I don't lean any particular way here... thoughts?

Reply:
Yup, 👍🏻 to keep the change. Sometimes there are little subtleties about when you can use one API or the other, and I was curious if I was missing something on a casual read-through. (for example: that a test started failing, because some context was missing when launched via `apply_create_message()`) But it sounds like I expected: just a cleaner way to do it.